### ISSUE TITLE
Another item in Scope combobox

### ISSUE THREAD

**Issue by klepetox** [VFPX/GoFish#232]
## 📝 Provide a description of the new feature

Is it possible to add item "Project in current dir" into Scope combobox?

**Comment by Jimrnelson** [comment]
I assume you mean you have only one project in the current dir?  (Option would not apply otherwise?)

**Comment by klepetox** [comment]
Yes, I have only one project in actual folder (project name is always as folder name -> hsp.pjx in c:\fox\hsp).

**Comment by Jimrnelson** [comment]
Added in #233

Version 7.0.33 - 2024-05-25

- Add new item "Project in CurDir" to scope type dropdown (#232)
- Fixed RegEx bug (#231)
